### DISCUSSION TITLE
Is there any way to use my own API key for a engine wich requres one?

### DISCUSSION THREAD

**Discussion by winux1** [searx/searx#2527] [General]
Hello!
I would like to use the flickr engine with my own API key on a public instance Is there any way to do it?

Thanks for your answer in advance!

**Comment by unixfox** [comment]
https://github.com/searx/searx/blob/master/searx/settings.yml#L408

**Reply by winux1** [reply]
Thanks for your help! But can I set api key on the client side?

**Reply by unixfox** [reply]
> Thanks for your help! But can I set api key on the client side?

You mean on a public instance without being the administrator of that instance? No it's not possible.

**Reply by winux1** [reply]
That’s sad. Thanks for your help again!

**Reply by kvch** [reply]
Do you mind opening an enhancement request for that? It seems like an interesting thing which we could support.

**Reply by winux1** [reply]
I will open one tomorrow
